### PR TITLE
etl: tracing: Add node client types for tracing

### PR DESCRIPTION
**Pull request by andrey-utila**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -49,4 +49,5 @@ venv/
 ENV/
 
 # etl
-/last_synced_block.txt
\ No newline at end of file
+/last_synced_block.txt
+.python-version
```
Comment (ika-utila) [inline .gitignore:53]: break line

File: blockchainetl/jobs/exporters/in_memory_item_exporter.py [hunk 2]
```diff
@@ -35,6 +35,8 @@ def export_item(self, item):
         if item_type is None:
             raise ValueError('type key is not found in item {}'.format(repr(item)))
 
+        print(self.items)
```
Comment (ika-utila) [inline blockchainetl/jobs/exporters/in_memory_item_exporter.py:38]: ?

File: .gitignore (hunk 1 above)
Comment (andrey-utila) [reply .gitignore]: done

File: ethereumetl/cli/stream.py [hunk 3]
```diff
@@ -98,6 +102,14 @@ def parse_entity_types(entity_types):
 
     return entity_types
 
+def parse_node_client_type(node_client_type):
+    # validate passed node client
+    if node_client_type not in NodeClientType.ALL_NODE_CLIENT_TYPES:
+        raise click.BadOptionUsage(
+            '--node-client', '{} is not an available node client type. supported clients are {}'
+                    .format(node_client_type, ','.join(NodeClientType.ALL_NODE_CLIENT_TYPES)))
```
Comment (ika-utila) [inline ethereumetl/cli/stream.py:110]: nit 
use F string

**Review by ika-utila — APPROVED** [review]

**Pull request merged**